client/modules/IDE/components/Header/MobileNav.jsx

```diff
@@ -0,0 +1,395 @@
+    <Nav>
+      <LogoContainer>
+        <Logo />
+      </LogoContainer>
+      <Title>
+        <h1>{title === project.name ? <ProjectName /> : title}</h1>
+        {project?.owner && title === project.name && (
+          <h5>by {project?.owner?.username}</h5>
+        )}
+      </Title>
+      <Options>
+        {user.authenticated ? (
+          <AccountMenu />
+        ) : (
+          <div>
+            <Link to="/login">
+              <IconButton icon={AccountIcon} />
+            </Link>
+          </div>
+        )}
+        {title === project.name ? (
+          <MoreMenu />
+        ) : (
+          <div>
+            <Link to={editorLink}>
+              <IconButton icon={EditorIcon} />
+            </Link>
+          </div>
+        )}
+      </Options>
+    </Nav>
+  );
+};
+
+const AccountMenu = () => {
+  const user = useSelector((state) => state.user);
+  const dispatch = useDispatch();
+
+  const { isOpen, handlers } = useMenuProps('account');
+
+  return (
+    <div>
+      <IconButton icon={AccountIcon} {...handlers} />
+      <ul className={isOpen ? 'opened' : ''}>
+        <ParentMenuContext.Provider value="account">
+          <li className="user">{user.username}</li>
+          <NavMenuItem href={`/${user.username}/sketches`}>
+            My Stuff
+          </NavMenuItem>
+          <NavMenuItem href="/account">Settings</NavMenuItem>
+          <NavMenuItem onClick={() => dispatch(logoutUser())}>
+            Log Out
+          </NavMenuItem>
+        </ParentMenuContext.Provider>
+      </ul>
+    </div>
+  );
+};
+
+const MoreMenu = () => {
+  // TODO: use selectRootFile selector
+  const rootFile = useSelector(
+    (state) => state.files.filter((file) => file.name === 'root')[0]
+  );
+  const language = useSelector((state) => state.preferences.language);
+
+  const dispatch = useDispatch();
+  const { t } = useTranslation();
+  const { newSketch, saveSketch } = useSketchActions();
+
+  const cmRef = useContext(CmControllerContext);
+
+  const { isOpen, handlers } = useMenuProps('more');
+
+  const [isLanguageModalVisible, setIsLanguageModalVisible] = useState(false);
+
+  function handleLangSelection(event) {
+    dispatch(setLanguage(event.target.value));
+    dispatch(showToast('Toast.LangChange'));
+    setIsLanguageModalVisible(false);
+  }
+
+  return (
+    <div>
+      {isLanguageModalVisible && (
+        <Overlay
+          // TODO: add translations
+          title="Select Language"
+          ariaLabel="Select Language"
+          closeOverlay={() => setIsLanguageModalVisible(false)}
+        >
+          <LanguageSelect>
+            <div>
+              {sortBy(availableLanguages).map((key) => {
+                const label = languageKeyToLabel(key);
+                return (
+                  <button
+                    className={classNames({
+                      'current-language': label === languageKeyToLabel(language)
+                    })}
+                    aria-label={label}
```

Comment:
I need to read up on ARIA best practices because I don't know if it's necessary to have an `aria-label` when it's the same as the text content of the button.  But we'll leave it for now.